lectures/inflation_history.md

```diff
@@ -13,64 +13,65 @@ kernelspec:
 
-Until the early 20th century, throughout much of the west, although price levels fluctuated from year to  year, they didn't have much of a trend.  
+Until the early 20th century, in many western economies, price levels fluctuated from year to year but didn't have much of a trend.  
 
-They tended to end a century near where  they started it.
+Often the price level ended a century near where they started.
 
 Things were different in the 20th century, as we shall see in this lecture.
 
-We'll indicater a widely believed explanation of this big difference -- countries' abandoning gold and silver standards in early twentieth century. 
-
-This lecture  sets the stage for some subsequent lectures about a theory that macroeconomists   economists use to think about determinants of the price level, namely, {doc}`this lecture <cagan_ree>` and 
-{doc}`this lecture <cagan_adaptive>`
-
+A widely believed explanation of this big difference is that countries' abandoning gold and silver standards in the early twentieth century. 
 
+```{tip}
+This lecture sets the stage for some subsequent lectures about a theory that macro economists use to think about determinants of the price level, namely, {doc}`cagan_ree` and {doc}`cagan_adaptive`
+```
 
 ## Four Centuries of Price Levels
 
-We begin by displaying    data  that originally appeared on page 35 of {cite}`sargent2002big` and that show  price levels for four "hard currency" countries from 1600 to 1914.
-
+We begin by displaying data that originally appeared on page 35 of {cite}`sargent2002big` that show price levels for four "hard currency" countries from 1600 to 1914.
 
 * France 
 * Spain (Castile)
 * United Kingdom
 * United States
 
-In the present context, the  phrase ''hard currency'' means that the countries were on a commodity-money standard:  money consisted of gold and silver coins that circulated at values largely determined by the weights of their gold and silver contents.
+In the present context, the  phrase "hard currency" means that the countries were on a commodity-money standard:  money consisted of gold and silver coins that circulated at values largely determined by the weights of their gold and silver contents.
 
 ```{note}
-Under a gold or silver standard, some money also consisted of "warehouse certificates" that represented paper claims on gold or silver coins. Bank notes issued by the government or private banks can be viewed as examples of such "warehouse certificates''.
+Under a gold or silver standard, some money also consisted of "warehouse certificates" that represented paper claims on gold or silver coins. Bank notes issued by the government or private banks can be viewed as examples of such "warehouse certificates".
 ```
 
+We start by installing the necessary Python packages.
+
+```{code-cell} ipython3
+:tags: [hide-output]
+!pip install xlrd
+```
 
-As usual, we'll start by importing some Python modules.
+We then import the Python modules we need.
```

Comment:
I think our convention is to have all imports and installs at the top.  Shall we move this and the previous cell up and modify the text?